### PR TITLE
Change width of column.title to avoid :before cover on it

### PR DESCRIPTION
**Pull request by orzyyyy**
#13467

### CODE REVIEW

File: components/table/style/index.less [hunk 1]
```diff
@@ -153,7 +153,7 @@
 
     .@{table-prefix-cls}-column-sorters {
       &:before {
-        position: absolute;
+        position: relative;
```
Comment (afc163) [inline components/table/style/index.less]: Why changing this?

Comment (zy410419243) [reply components/table/style/index.less]: If not, `column` will cover on the `title`, so `Popover` can't show in #13467

Comment (zy410419243) [reply components/table/style/index.less]: So what can I do with it.

Comment (afc163) [reply components/table/style/index.less]: 这样会改变现在的 hover 样式。

image

你可以对比一下线上和改动后的效果。

Comment (afc163) [reply components/table/style/index.less]: 相当于把这个伪元素删了。

Comment (zy410419243) [reply components/table/style/index.less]: 讲道理，鼠标移到上下箭头再显示 `排序` 才合理的

中间空的这块，如果传进来 `ReactNode`， 如果有 `title` 这个属性，那显示出来，其他时候不显示貌似也没什么问题啊，反正这里最重要的应该是 `排序` 这个功能才对吧...

Comment (afc163) [reply components/table/style/index.less]: 我知道问题所在，我也同意要改。

现在的问题是去掉这个伪元素会影响 hover 的设计。

需要在保证这个设计的情况下，修复 title 的 bug。

Comment (afc163) [reply components/table/style/index.less]: 线上 hover 效果：

image

本 PR hover 效果：

image

**Comment by admire7444** [comment]
table的排序也把在columns的title中渲染的ReactNode的功能给覆盖掉了
例如我使用一个Ellipsis作为columns的title用来解决title中文字过长的问题，但是同时开启了表格的排序功能后，Ellipsis就被排序给覆盖住了，鼠标移到上面显示的就是"排序"而不是弹出title中的全部文字了

**Comment by zy410419243** [comment]
avatar

可能这是我能想到的最浪漫的事了

**Comment by zy410419243** [comment]
> ```
>     console.error node_modules/react-dom/cjs/react-dom.development.js:506
>       Warning: Invalid value for prop `title` on <div> tag. Either remove it from the element, or pass a string or number value to keep it in the DOM. For details, see https://fb.me/react-attribute-behavior
>           in div (created by LocaleReceiver)
>           in th (created by TableHeaderRow)
>           in tr (created by TableHeaderRow)
>           in TableHeaderRow (created by Connect(TableHeaderRow))
>           in Connect(TableHeaderRow) (created by TableHeader)
>           in thead (created by TableHeader)
>           in TableHeader (created by BaseTable)
>           in table (created by BaseTable)
>           in BaseTable (created by Connect(BaseTable))
>           in Connect(BaseTable) (created by BodyTable)
>           in div (created by BodyTable)
>           in BodyTable (created by ExpandableTable)
>           in div (created by ExpandableTable)
>           in div (created by ExpandableTable)
>           in ExpandableTable (created by Connect(ExpandableTable))
>           in Connect(ExpandableTable) (created by Table)
>           in Provider (created by Table)
>           in Table (created by LocaleReceiver)
>           in LocaleReceiver (created by Table)
>           in div (created by Spin)
>           in AnimateChild (created by Animate)
>           in div (created by Animate)
>           in Animate (created by Spin)
>           in Spin (created by Table)
>           in div (created by Table)
>           in Table (created by WrapperComponent)
>           in WrapperComponent
> ```
> master 上现在还有这个问题。

这个是怎么做到的...

**Pull request closed without merging**